llmfoundry/callbacks/eval_gauntlet_callback.py

```diff
@@ -43,6 +59,7 @@ class EvalGauntlet(Callback):
         rescale_accuracy (bool): Flag determining whether to rescale the accuracy on each benchmark
                                  by (1-random_baseline_accuracy) before aggregating. Using this ensures that all benchmarks max out at 1.0.
         benchmark_sizes (Optional[dict]): Optional data on benchmark sizes, used when not relying on equal weighting.
+        averages (Optional[dict]): Optional specification on how to which categories to average together and the name under which to report each average.
```

Comment:
not sure what this is supposed to say exactly, think it has a typo or two. please update and also include an example of what this looks like